packages/contracts/src/Multisig.sol

```diff
@@ -152,6 +166,18 @@ contract Multisig is
         emit MembersAdded({members: _members});
 
         _updateMultisigSettings(_multisigSettings);
+
+        _setTargetConfig(_targetConfig);
+    }
+
+    /// @notice Reinitializes the TokenVoting after an upgrade from a previous protocol version.
+    /// @param _fromBuild The build version number of the previous implementation contract this upgrade is transitioning from.
+    /// @param _initData The initialization data to be passed to via `upgradeToAndCall` (see [ERC-1967](https://docs.openzeppelin.com/contracts/4.x/api/proxy#ERC1967Upgrade)).
+    function initializeFrom(uint16 _fromBuild, bytes calldata _initData) external reinitializer(2) {
+        if (_fromBuild < 3) {
```

Comment:
I noticed that in OSX, they are checking the entire version. However, it seems like this may not be necessary for plugins, since upgrading between different releases (e.g., from 1.5 to 2.3) isn’t allowed as far as I understand.

I'd like to hear your thoughts on this.

Reply:
Yes, We don't use semantic versioning for plugins and can not be upgraded between different `releases`. So all good.